doc-content/jbpm-docs/src/main/asciidoc/CoreEngine/manage-execution-errors-proc.adoc

```diff
@@ -0,0 +1,14 @@
+[id='manage-execution-errors-proc-{context}']
+= Managing execution errors
+
+By definition, every process error that is detected and stored is unacknowledged and must be handled by someone or something (in case of automatic error recovery). You can view a filtered list of errors that were or were not acknowledged. Acknowledging an error saves the user information and time stamp for traceability.
+
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Manage* -> *Execution Errors*.
```

Comment:
```suggestion
. In {CENTRAL}, select *Menu* -> *Manage* -> *Execution Errors*.
```

Reply:
I l iike to say what happens after you that. So.. The Manage page appears.

Now you have context for the step.

Reply:
Probably better but I think we need to be consistent, and do this for a lot of steps in many parts of this same document (including subsequent steps here). Afraid I can't pull that off right now. Something to do in quality improvement - perhaps same run as "clarify optional steps"